client/my-sites/site-settings/date-time-format/date-time-format-options.jsx

```diff
@@ -0,0 +1,272 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { includes, startsWith } from 'lodash';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import ExternalLink from 'components/external-link';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormInput from 'components/forms/form-text-input';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import FormSelect from 'components/forms/form-select';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import SectionHeader from 'components/section-header';
+import { phpToMomentDatetimeFormat } from 'lib/formatting';
+import wrapSettingsForm from '../wrap-settings-form';
+
+export class DateTimeFormatOptions extends Component {
+	dateFormatOption() {
+		const {
+			fields: { date_format, timezone_string },
```

Comment:
What if you destructured these to camelCase names?

`{ date_format: dateFormat, timezone_string: timezoneString }`